content/docs/about/kubeflow.md

```diff
@@ -29,11 +29,25 @@ Ultimately, we want to have a set of simple manifests that give you an easy to
 use ML stack _anywhere_ Kubernetes is already running, and that can self 
 configure based on the cluster it deploys into.
 
+## What is Kubeflow?
+
 Kubeflow is *the machine learning toolkit for Kubernetes*. 
 
-## What is Kubeflow?
+It's a collection of configuration files and scripts that you download,
```

Comment:
I prefer the existing description which provides a much higher level description of what Kubeflow is. That description allows underlying details of Kubeflow as a platform to evolve without requiring changes to what defines it.

Reply:
+1. Also Kubeflow has more components than the main repository (which is where the config files and scripts are). For example we also have TF training operators and serving.

Reply:
@abhi-g @richardsliu thank you for your review. 
Should we add this information (from v0.2) bellow or just leave "What is Kubeflow" as it is?

Reply:
@andreyvelich I looked at the git issue from Sarah that you referenced. It seems like a more appropriate change would be to make a PR in the v0.2 branch of the docs to make this section identical to this paragraph in master.

Reply:
@andreyvelich I looked at the git issue from Sarah that you referenced. It seems like a more appropriate change would be to make a PR in the v0.2 branch of the docs to make this section identical to this paragraph in master.

Reply:
@abhi-g Change it in v0.2 branch